composer.json

```diff
@@ -14,7 +14,8 @@
     ],
     "require": {
         "php": "^7.0",
-        "psr/log": "^1.0.1"
+        "psr/log": "^1.0.1",
+        "rollbar/rollbar": "^1.3"
```

Comment:
This should probably go into `suggest` section of `composer.json` since not all Monolog users need the `rollbar/rollbar` dependency.